src/test/java/org/opensearch/geospatial/index/query/xyshape/XYShapeQueryIT.java

```diff
@@ -0,0 +1,272 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.geospatial.index.query.xyshape;
+
+import static org.hamcrest.Matchers.equalTo;
+import static org.opensearch.common.xcontent.ToXContent.EMPTY_PARAMS;
+import static org.opensearch.geospatial.GeospatialTestHelper.randomLowerCaseString;
+import static org.opensearch.index.query.AbstractGeometryQueryBuilder.DEFAULT_SHAPE_FIELD_NAME;
+import static org.opensearch.test.hamcrest.OpenSearchAssertions.assertHitCount;
+import static org.opensearch.test.hamcrest.OpenSearchAssertions.assertSearchResponse;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.Matchers;
+import org.opensearch.action.search.SearchResponse;
+import org.opensearch.common.geo.GeoJson;
+import org.opensearch.common.geo.ShapeRelation;
+import org.opensearch.common.settings.Settings;
+import org.opensearch.geometry.Circle;
+import org.opensearch.geometry.LinearRing;
+import org.opensearch.geometry.MultiPolygon;
+import org.opensearch.geometry.Point;
+import org.opensearch.geometry.Polygon;
+import org.opensearch.geometry.Rectangle;
+import org.opensearch.geospatial.GeospatialRestTestCase;
+import org.opensearch.geospatial.index.mapper.xyshape.XYShapeFieldMapper;
+import org.opensearch.search.SearchHit;
+
+public class XYShapeQueryIT extends GeospatialRestTestCase {
+
+    public static final String INDEXED_SHAPE_FIELD = "indexed_shape";
+    public static final String SHAPE_INDEX_FIELD = "index";
+    public static final String SHAPE_ID_FIELD = "id";
+    public static final String SHAPE_INDEX_PATH_FIELD = "path";
```

Comment:
do we need those constants as public?

Reply:
Good catch. I created these constants using IDE, didn't check their default access level.